src/ImportDefinitionsBundle/Importer/Importer.php

```diff
@@ -396,7 +396,7 @@ private function getObjectForPrimaryKey(DefinitionInterface $definition, $data)
                 $obj = $objectData[0];
             }
 
-            if ($obj === null) {
+            if (null === $obj) {
```

Comment:
may the force be with you my young Padawan

Reply:
Always master!